tap_harvest/__init__.py

```diff
@@ -273,8 +292,7 @@ def map_invoice_payment(payment):
                       path="invoice_messages",
                       with_updated_since=False,
                       map_handler=map_invoice_message,
-                      for_each_handler=for_each_invoice_message,
-                      date_fields=["send_reminder_on"])
```

Comment:
Should this also be removed from `sync_endpoint` or is it still used?

Reply:
I mean the function that's being called with `date_fields`, this one: https://github.com/singer-io/tap-harvest/blob/master/tap_harvest/__init__.py#L152

Though it looks like there are other places in the file where `date_fields` is used, so this is fine.